cache/test_cache.py

```diff
@@ -94,4 +95,72 @@ class TestCacheReplicated(TestCacheDisabled):
     # Replicated mode.
     cache_mode = 2
 
-# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+class H2Cache(tester.TempestaTest):
+    clients = [{
+        'id': 'deproxy',
+        'type': 'deproxy_h2',
+        'addr': "${server_ip}",
```

Comment:
I believe would be better to use `${tempesta_ip}`, because tempesta and backend can be on different hosts.